test/it/ArrivalSpecWithIleQueryDisabled.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.time.temporal.ChronoUnit
+import java.time.{LocalDate, LocalDateTime, LocalTime, ZoneOffset}
+
+import forms.ConsignmentReferences
+import models.cache.ArrivalAnswers
+import play.api.test.Helpers._
+
+class ArrivalSpecWithIleQueryDisabled extends IntegrationSpec with IleQueryDisabled {
```

Comment:
I am not convinced about separate Spec just to test different configuration. I am mostly concerned about approach like this, having in mind that we will have to clear everything after the feature flag is not used.
I'd probably put both scenarios into single test suite, but changed the way `ileQueryFeatureConfiguration` value is obtained, to make it parameterized.